hea/urls.py

```diff
@@ -67,6 +69,8 @@
 router.register(r"currency", CurrencyViewSet)
 router.register(r"unitofmeasure", UnitOfMeasureViewSet)
 router.register(r"classifiedproduct", ClassifiedProductViewSet)
+router.register(r"user", UserViewSet)
+router.register(r"heaprofile", HeaProfileViewSet)
```

Comment:
@girumb I think i would prefer if this was `/userprofile`, and the model was `UserProfile`, etc. It is in `common` so it is not supposed to be specific. It's too late for FDW now, but I would like a model that we can just copy without changes into the next application we build.